src/Symfony/Component/Ldap/Entry.php

```diff
@@ -22,9 +22,24 @@ class Entry
     public function __construct(string $dn, array $attributes = array())
     {
         $this->dn = $dn;
+        $attributes = $this->resetValueIndex($attributes);
         $this->attributes = $attributes;
     }
 
+    /**
+     * Resets the index for array value
+     *
+     * @param array $attributes
+     *
+     * @return array
+     */
+    private function resetValueIndex($attributes = array())
+    {
+        return array_map(function ($o) {
```

Comment:
is $o an array type ?

What does it mean ?

Reply:
$o should be an array, for the meaning it is inside the callback function it can be whatever we want. Is there any specification for callback functions in Symfony?